math.mod/math.c

```diff
@@ -131,6 +141,8 @@ double bbTrunc( double x ){
 #define ceilf ceil
 #define roundf round
 #define truncf trunc
+#define deg2radf deg2rad
+#define rad2degf rad2deg

```

Comment:
This is not required.

Reply:
Can you tell why? 

Or better: why is it needed for the other entries? I simply tried to do as done for all the other functions.

Reply:
What I mean is - if I remove these lines, should I remove the other ones too?
```

#define sqrtf sqrt
#define sinf sin
#define cosf cos
#define tanf tan
#define asinf asin
#define acosf acos
#define atanf atan
#define atan2f atan2
#define sinhf sinh
#define coshf cosh
#define tanhf tanh
#define expf exp
#define floorf floor
#define logf log
#define log10f log10
#define ceilf ceil
#define roundf round
#define truncf trunc
#define deg2radf deg2rad
#define rad2degf rad2deg
```

Reply:
I removed the blamed lines - just drop a line if I should remove the other define-lines (see above) too.

If the PR is then OK, do not forget to "squash" the commits (to remove the extra commit from git history).